site/man/rabbitmq-streams.8.html

```diff
@@ -232,6 +232,16 @@ <h3 class="Ss" title="Ss" id="Stream_plugin"><a class="permalink" href="#Stream_
       <var class="Ar" title="Ar">consumerinfoitem</var> can take any value from
       the list that follows:
     <dl class="Bl-tag">
+      <dt><var class="Ar" title="Ar">active</var></dt>
+      <dd>Boolean indicating whether the consumer is active or not.</dd>
+      <dt><var class="Ar" title="Ar">activity_status</var></dt>
+      <dd>Consumer activity status; one of:
```

Comment:
Instead of "one of" How about starting a new sentence saying: The value can be one of the following:
GLOBAL comment for anywhere below where it applies